setup.cfg

```diff
@@ -8,3 +8,7 @@ minversion =
     3.0.2
 testpaths =
     ./tests/
+
+[build_sphinx]
+project = 'cytominer_database'
+version = 0.3.3
```

Comment:
also add this
`copyright = 2019, Allen Goodman, Claire McQuin, Shantanu Singh, Gregory Way` and remove [these](https://github.com/cytomining/cytominer-database/blob/61af4a4ce8464616de9e7c057db366beda67c70a/doc/conf.py#L51-L52) lines

Reply:
done in 11e49e58db5e35e05409a5b7dec3c461235acb8e